docs/static/security/es-security.asciidoc

```diff
@@ -0,0 +1,77 @@
+[discrete]
+[[es-security-on]]
+== {es} security on by default
+
+{es} {ref}/configuring-stack-security.html[security is on by default] starting in 8.0.
+{es} generates its own default self-signed Secure Sockets Layer (SSL) certificates at startup. 
+
+For an on-premise {es} cluster, {ls} must establish a secure connection using the self-signed SSL certificate before it can transfer data.  
+{ess} uses standard publicly trusted certificates, and therefore setting a cacert is not necessary.
+
+.Hosted {ess} simplifies security
+[NOTE]
+=====
+Our hosted {ess} on Elastic Cloud simplifies safe, secure communication between {ls} and {es}. 
+When you configure the elasticsearch output plugin to use <<plugins-outputs-elasticsearch-cloud_id,`cloud_id`>> with either the <<plugins-outputs-elasticsearch-cloud_auth,`cloud_auth` option>> or the <<plugins-outputs-elasticsearch-api_key,`api_key` option>>, no additional SSL configuration steps are needed.
+
+Examples:
+
+* `output {elasticsearch { cloud_id => "<cloud id>" cloud_auth => "<cloud auth>" } }`
+* `output {elasticsearch { cloud_id => "<cloud id>" api_key => "<api key>" } }``
+
+For more details, check out the
+{logstash-ref}/connecting-to-cloud.html[Logstash-to-Cloud documentation].
+
+{ess-leadin-short}
+=====
+
+[discrete]
+[[es-security-onprem]]
+=== Secure communication with an on-premise {es} cluster
+
+If you are running {es} on your own hardware, you need to complete a few more steps to establish secure communication between {ls} and {es}.
+You need to: 
+
+* Copy the self-signed CA certificate from {es} and save it to {ls}
+* Configure the elasticsearch-output plugin to use the certificate
+
+[discrete]
+[[es-sec-copy-cert]]
+==== Copy and save the certificate
+
+On on-premise {es} cluster generates a self-signed CA and creates its own SSL certificates when it starts.
+{ls} needs its own copy of the self-signed certificate from the {es} cluster so that it can validate the certificate presented by {es}.
+
+Copy the {ref}/configuring-stack-security.html#stack-security-certificates[self-signed CA certificate] from the {es} `config/certs` directory.
+
+Save it to a location that Logstash can access, such as `config/certs` on the {ls} instance. 
+
+/////
+ToDo: 
+Verify location. 
+I don't see an existing config/certs folder. 
+If this is the location we'd like users to use, we should stub out a folder in https://github.com/elastic/logstash/tree/main/config. 
+As always, there's a definite argument for consistency across deployments. 
+/////
+
```

Comment:
Should we stub out a `/certs` folder in https://github.com/elastic/logstash/tree/main/config?